.projenrc.ts

```diff
@@ -1059,8 +1059,8 @@ const toolkitLib = configureProject(
     description: 'AWS CDK Programmatic Toolkit Library',
     srcdir: 'lib',
     deps: [
-      cloudAssemblySchema,
-      cloudFormationDiff,
+      cloudAssemblySchema.customizeReference({ exactVersion: true }),
+      cloudFormationDiff.customizeReference({ exactVersion: true }),
```

Comment:
That's not right. Toolkit Library intentionally uses dependency ranges. In fact it should probably be wider and declare the actual minimal version and not just `^current`.